### ISSUE TITLE
Can you please suggest how much gpu memory should I use in order to complete the training? mine is a 4g gpu, and with that config set in sess, it ran out of resource every time I try. Thanks.

### ISSUE THREAD

**Issue by Flatheadman** [taki0112/SENet-Tensorflow#7]

**Comment by Flatheadman** [comment]
Never mind. I changed batch size to 32 and test iteration to 100 and it worked fine on my 4g gpu :) 
BTW: accuracy will improve significantly at epoch 30, 60 and 90, be patient :)
